pages/blocks/learnmoresmart/learnmoresmart.css

```diff
@@ -0,0 +1,76 @@
+main div.block.learnmoresmart {
+  visibility: unset;
+}
+
+main .section-wrapper.learnmoresmart-container > div {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.learnmoresmart {
+  padding: 2em 0em;
+}
+
+
+.learn-smart-block {
+  margin-bottom: 3em;
+}
+
+.learn-smart-block:last-of-type {
+  margin-bottom: 0em;
+}
+
+.learn-image {
+  padding-bottom: 56.25%;
+  position: relative;
+}
+
+.learn-image img {
+  object-fit: cover;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  left: 0;
+}
+
+.learn-title {
+  font-weight: 800;
+  font-size: 20px;
+  margin: 0;
+  margin-top: 0.5em;
+}
+
+.learn-copy {
+  font-size: 17px;
+  margin-top: 0.5em;
+}
```

Comment:
You should specify .learnmoresmart class as well to avoid potential issue with other block in case they use same class name.